ban/commands/ignsna.py

```diff
@@ -0,0 +1,111 @@
+import glob
+import os
+
+from peewee import IntegrityError
+
+from ban.commands import command, report
+from ban.core.models import (HouseNumber, Locality, Municipality, Position,
+                             Street, PostCode)
+from .helpers import iter_file, session, batch
+
+__namespace__ = 'import'
+
+
+@command
+def ignsna(path, **kwargs):
+    """Import from IGN/Laposte BDUNI
+    :param path: directory location of the files hsp7xxxx.ai, hsv7xxxx.ai and hsw4xxxx.ai"""
+
+    municipality_zipcode_file = glob.glob(os.path.join(path, 'hsp7*.ai'))
+    # street_file = glob.glob(os.path.join(path, 'hsv7*.ai'))
+    # number_file = glob.glob(os.path.join(path, 'hsw4*.ai'))
+
+    if municipality_zipcode_file is not None:
+        max_value = get_max_line(municipality_zipcode_file[0])
+        lines = list(get_lines(municipality_zipcode_file[0]))
```

Comment:
No, because file buffer will be consumed by len(list(lines)), and thus will be empty when `batch` will try to iterate again on `lines`.
I need to find a better way to compute max_values :s